javascript/packages/orchestrator/src/providers/k8s/substrateCliArgsHelper.ts

```diff
@@ -25,5 +25,5 @@ export const getCliArgsVersion = async (
 
   return logs.includes("--ws-port <PORT>")
```

Comment:
We should check here if the flag ` --insecure-validator-i-know-what-i-do` is present in the output to ensure that is `V2`, since could be that people are using not the latest version of polkadot.